src/App.tsx

```diff
@@ -20,12 +36,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={handleChange}
```

Comment:
you should use setQuery in this taks (use hooks, not class elements)

Reply:
I don't use class elements, handleChange is only a callback which I moved on from onChange.